### ISSUE TITLE
icu: incorrect options for 'data_packaging'

### ISSUE THREAD

**Issue by chrisguiney** [bincrafters/community#40]
## Description
ICU lists `"shared", "static", "files", "archive"` as the possible values

`shared` is incorrect, the correct value is `library`

### package
icu/59.1@bincrafters/stable

### Extra information
From the configure script:
```
  --with-data-packaging     specify how to package ICU data. Possible values:
        files    raw files (.res, etc)
        archive  build a single icudtXX.dat file
        library  shared library (.dll/.so/etc.)
        static   static library (.a/.lib/etc.)
        auto     build shared if possible (default)
           See http://userguide.icu-project.org/icudata for more info.
```

* Conan version: **v0.30.3**
* Operating System: **Linux**
* Compiler+version: **gcc-7.2**
* Docker image: **debian:latest**

**Comment by sigmoidal** [comment]
I always use "archive" so I didn't notice this. This is due to an error in the ICU web page mentioned http://userguide.icu-project.org/icudata. I filled a bug report to update the docs upstream. 

Thanks I will amend this soon once 60.2 is released.

**Comment by sigmoidal** [comment]
this has been addressed, the new recipe for 60.2 does pass the correct options.
